dll/win32/comctl32/header.c

```diff
@@ -1948,13 +1948,14 @@ HEADER_MouseMove (HEADER_INFO *infoPtr, LPARAM lParam)
                     
                     if (nWidth < 0) nWidth = 0;
                     infoPtr->items[infoPtr->iMoveItem].cxy = nWidth;
+                    InvalidateRect(infoPtr->hwndSelf, &lpItem->rect, FALSE);
```

Comment:
Unfortunately as of now, this code is still synced with Wine so your changes must be guarded using `#ifdef` and `#ifndef` statements.

```suggestion
#ifdef __REACTOS__
                    InvalidateRect(infoPtr->hwndSelf, &lpItem->rect, FALSE);
#endif
```

Reply:
Is this call needed, since there is again an invalidate on the same window below?

Reply:
For some reason not having that call produces redraw issues in the not text portion of the tab.

https://github.com/reactos/reactos/assets/65135562/ac5e8849-acad-45ff-901f-7e9cd8479dec